src/dune_rules/melange_rules.ml

```diff
@@ -139,8 +154,14 @@ let add_rules_for_entries ~sctx ~dir ~expander ~dir_contents ~scope
         build_js ~dir ~loc:(Some loc) ~pkg_name ~module_system:mel.module_system
           ~dst_dir ~obj_dir ~sctx ~build_dir ~lib_deps_js_includes m)
   in
-  let ctx = Super_context.context sctx in
-  let stdlib_dir = ctx.Context.stdlib_dir in
+  let* stdlib_dir =
+    let* melc = Super_context.resolve_program sctx ~dir ~loc:None "melc" in
```

Comment:
should we store the `melc` program name in a variable somewhere? It exists both here and in `"src/dune_rules/melange_rules.ml"` now.

Reply:
Fixed now.